Tests/PythonTests/test_cards.py

```diff
@@ -0,0 +1,216 @@
+"""
+Copyright (c) 2019 Chris Ohk, Youngjoong Kim, SeungHyun Jeon
+
+We are making my contributions/submissions to this project solely in our
+personal capacity and are not conveying any rights to any intellectual
+property of any third parties.
+"""
+
+import pyRosetta
+
+def test_all_cards():
+	cards = pyRosetta.Cards.all_cards()
+
+	assert len(cards) == 7735
+
+def test_all_standard_cards():
+	cards = pyRosetta.Cards.all_standard_cards()
+
+	for card in cards:
+		assert card.is_standard_set() == True
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comparison to True should be 'if cond is True:' or 'if cond:'](https://app.codacy.com/app/utilForever/RosettaStone/pullRequest?prid=4593740)